.github/workflows/make-with-lints.sh

```diff
@@ -0,0 +1,3 @@
+make |& \
```

Comment:
We could move the option in here, and add the interpreter. We can also set the executable bit on the file in the git repository to avoid having to chmod the script before running it.
```suggestion
#!/bin/bash
set -o pipefail
make |& \
```

Reply:
Since we are now invoking a separate script, I think we need to set the `-e` option as well in order to propagate failures back out to the parent shell. I think this is masking the unconditional failure introduced by `(! grep -v 'Warning: Found SVG ...')`.

Reply:
Added `set -e`. However I tested this by modifying the match string, and it caused the action to fail as expected.